insights/specs/__init__.py

```diff
@@ -151,6 +151,7 @@ class Specs(SpecSet):
     foreman_ssl_access_ssl_log = RegistryPoint(filterable=True)
     foreman_rake_db_migrate_status = RegistryPoint()
     foreman_tasks_config = RegistryPoint(filterable=True)
+    freeipa_healthcheck_log = RegistryPoint(filterable=True)
```

Comment:
From the definition of the parser `FreeIPAHealthCheckLog `, the base parser `JSONParser` requires the input file being in good JSON format. 

However, here, the spec is marked as `filterable=True`, which means the collected `/var/log/ipa/healthcheck/healthcheck.log` file must be filtered by some specified string (a whitelist). And its JSON format must be broken and results in Parser failure at last.

That is, once decided to use `JSONParser`, the content should NOT be filtered (the `filterable=Ture` should be removed). 
But if it must be filtered (e.g. to avoid collecting huge size file, or sensitive info), better change to use `LogFileOutput` or others as the base class.

Reply:
Thanks for the explanation, I've removed filterable=True.